packages/autocertifier-client/src/AutoCertifierClient.ts

```diff
@@ -0,0 +1,172 @@
+import { EventEmitter } from 'eventemitter3'
+import { IAutoCertifierService } from './proto/packages/autocertifier/protos/AutoCertifier.server'
+import { SessionIdRequest, SessionIdResponse } from './proto/packages/autocertifier/protos/AutoCertifier'
+import { ServerCallContext } from '@protobuf-ts/runtime-rpc'
+import { filePathToNodeFormat } from '@streamr/utils'
+import { RestClient } from './RestClient'
+import { CertifiedSubdomain } from './data/CertifiedSubdomain'
+import fs from 'fs'
+import path from 'path'
+import * as forge from 'node-forge'
+import { Logger } from '@streamr/utils'
+
+interface AutoCertifierClientEvents {
+    updatedSubdomain: (domain: CertifiedSubdomain) => void
+}
+
+const logger = new Logger(module)
+
+export const AUTOCERTIFIER_SERVICE_ID = 'system/auto-certificer'
```

Comment:
```suggestion
export const SERVICE_ID = 'system/auto-certificer'
```

Reply:
AUTOCERTIFIER_SERVICE_ID is an export used by the `DHT` and `autocertifier` packages

Reply:
I think it makes more sensdes here to keep the name as is. It is a lot easier to search for the correct id from the code if the naming is consistent. Also the id is exported by the client so it makes makes sense to specify the application in the name